hiedb.el

```diff
@@ -134,10 +134,12 @@
              cmdArgs)
       (read-only-mode 1)
       (display-buffer log-buffer
-                      '(display-buffer-pop-up-window . ((side . top)
-                                                        (window-height . 5)
-                                                        (mode . (special-mode))
-                                                        ))))))
+                      '((display-buffer-in-side-window . ((side . top)
```

Comment:
I'm wondering if we'll get a bit further with moving to `compilation-start` from `compile.el` for creating the buffer from the output of `hiedb`